recipes/gtest/all/conanfile.py

```diff
@@ -72,22 +72,27 @@ def package_id(self):
         del self.info.options.no_main
 
     def package_info(self):
-        if self.options.build_gmock:
-            gmock_libs = ['gmock', 'gtest'] if self.options.no_main else ['gmock_main', 'gmock', 'gtest']
-            self.cpp_info.libs = ["{}{}".format(lib, self._postfix) for lib in gmock_libs]
-        else:
-            gtest_libs = ['gtest'] if self.options.no_main else ['gtest_main' , 'gtest']
-            self.cpp_info.libs = ["{}{}".format(lib, self._postfix) for lib in gtest_libs]
-
+        self.cpp_info.name = "GTest"
+        self.cpp_info.components["libgtest"].names["cmake_find_package"] = "gtest"
+        self.cpp_info.components["libgtest"].libs = ["gtest{}".format(self._postfix)]
         if self.settings.os == "Linux":
-            self.cpp_info.system_libs.append("pthread")
+             self.cpp_info.components["libgtest"].system_libs.append("pthread")
```

Comment:
be careful with extra spaces
```suggestion
            self.cpp_info.components["libgtest"].system_libs.append("pthread")
```